### ISSUE TITLE
OpenMW doesn't use DGPU with Hybrid graphics enabled

### ISSUE THREAD

**Issue by calhix** [pop-os/pop#2039]
**Distribution (run `cat /etc/os-release`):** 
Pop_OS! 21.04
 
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):** 
OpenMW-Nightly

**Issue/Bug Description:**
OpenMW-Nightly doesn't use the DGPU when Hybrid Graphics is enabled. This is confirmed with nvidia-smi while application is running. After switching to Nvidia Graphics in the power menu, it then uses the DGPU. This doesn't happen with any other game I have.

**Steps to reproduce (if you know):**
With Hybrid graphics enabled, start openmw-nightly, FPS is most likely much lower. Run nvidia-smi and confirm that DGPU is not being used.

**Expected behavior:**
OpenMW-Nightly should use DGPU with Hybrid Graphics enabled.

**Other Notes:**
I haven't tested with OpenMW stable. I can open an issue with the openmw team on gitlab if the issue could be corrected with them, if so, please relay any information that could be helpful and I will pass it on, I'm pretty sure this wasn't an issue with an older Nvidia driver but I'm not completely sure. Currently using the Nvidia 470.86 driver with a GTX 1650 and AMD Ryzen 7 4800HS.

**Comment by mmstick** [comment]
You chose to launch the game with dedicated graphics? It's not an automatic process on Linux.

**Comment by calhix** [comment]
> You chose to launch the game with dedicated graphics? It's not an automatic process on Linux.

I guess I've been lucky since every game on steam and lutris uses DGPU when using hybrid graphics. When right-clicking and selecting "launch using dedicated graphics card" throws the error "failed to create SDL window: Couldn't find matching GLX visual"

**Comment by mmstick** [comment]
Steam's desktop entry has `PrefersNonDefaultGPU=true` set, so it automatically applies this to games launched through Steam.

**Comment by calhix** [comment]
Makes sense, same for lutris i presume? So I guess if my preference is to stay on hybrid graphics, I should just launch it through steam/maybe lutris?
